crates/compilers/src/compilers/zksolc/input.rs

```diff
@@ -65,10 +70,15 @@ impl CompilerInput for ZkSolcVersionedInput {
 
 /// Input type `zksolc` expects.
 #[derive(Clone, Debug, Serialize, Deserialize)]
+#[serde(rename_all = "camelCase")]
 pub struct ZkSolcInput {
     pub language: SolcLanguage,
     pub sources: Sources,
     pub settings: ZkSettings,
+    #[serde(default, skip_serializing_if = "HashSet::is_empty")]
```

Comment:
are we keeping these fields here in the PR?

Reply:
No we should move them as done [here](https://github.com/Moonsong-Labs/compilers/commit/b04a92034e978a1626fd81b2a9fdc335f42e38e8) once https://github.com/Moonsong-Labs/compilers/pull/35 gets merged

Reply:
Oh so after 1.5.7, ok